package.json

```diff
@@ -1,108 +1,43 @@
 {
-  "name": "fd-vue-webapp",
-  "version": "1.0.0",
-  "description": "A Vue.js web application for Freedomotic",
-  "author": "Freedomotic Team",
+  "name": "freedomotic",
+  "version": "0.0.1",
+  "description": "A Quasar Framework app",
+  "productName": "Freedomotic Web App",
+  "cordovaId": "org.cordova.quasar.app",
+  "author": "atanda rasheed <atandarash@gmail.com>",
```

Comment:
please, avoid personal references here. We are a team of people, so your name should go in the contributor list. :smile:

Reply:
I'm so sorry (:disappointed:) I didn't see that.

Reply:
:+1: